web/themes/custom/usagov/sass/_header.scss

```diff
@@ -100,13 +101,22 @@
     margin-right: 2em;
   }
 
-  @include u-text('white');
   text-decoration: none;
   padding: 10px 20px;
-  @include u-bg('blue-80v');
   line-height: .9;
   font-size: .93rem;
   margin: 0;
+
+  @include u-text('white');
+  @include u-bg('blue-80v');
+
+  &:hover {
+    border: 2px solid color('cyan-50v');
+    padding: 8px 18px;
```

Comment:
To avoid layout shift on hover, remove the padding here, then in the selector right above this, .usa-header a.language-link, add this rule `border: 2px solid transparent`. This is how I avoided the same issue on wizard option hover.